pkg/operator/operator.go

```diff
@@ -80,6 +84,11 @@ func New(context *clusterd.Context) *Operator {
 // Run the operator instance
 func (o *Operator) Run() error {
 
+	namespace := os.Getenv(k8sutil.PodNamespaceEnvVar)
+	if namespace == "" {
+		return errors.New("Rook system namespace is not provided. Expose it via downward API in the rook operator manifest file")
```

Comment:
maybe add what the expected env var name is to this error message using `fmt.Errorf`

Reply:
done